dingo_test.go

```diff
@@ -313,3 +313,30 @@ func TestInjector_InitModules(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Error(t, injector.InitModules(new(testInjectInvalid)))
 }
+
+type TestInjectStructRecInterface interface {
+	TestXyz()
+}
+
+type testInjectStructRecStruct struct {
+	test string
+}
+
+func (t testInjectStructRecStruct) TestXyz() {
+
+}
+
+func (t testInjectStructRecStruct) Inject(test string) {
+	t.test = test
```

Comment:
As the assigment has no effect, can you remove it?

Reply:
Done. Had to remove the `test` field within `testInjectStructRecStruct` struct too. Otherwise there is another linting issue with `unused` field.